### PR TITLE
Dutch translations - iTop 2.6 (core, UI)

### PR DESCRIPTION
**Pull request by jbostoen**
* quick first translation new strings in core and UI files.
* typo in English dictionary
* incorporated suggestions by Hipska

I am not planning to use this pull request to work on other files, since I originally just wanted to translate the new stuff. :)

### CODE REVIEW

**Comment by Molkobain** [comment]
Hi @jbostoen ,

So this is the version you aggred with @Hipska , can we merge it?

**Comment by jbostoen** [comment]
It contains all the things we discussed so far and agreed upon.

I'm sure in the future if any of us have some time it could be further improved, but this is already a big step in the right direction.

File: dictionaries/en.dictionary.itop.ui.php [hunk 1]
```diff
@@ -673,7 +673,7 @@
...
-	'UI:Query:UrlV1' => 'The list of fields has been left unspecified. The page <em>export-V2.php</em> cannot be invoked without this information. Therefore, the URL suggested here below points to the legacy page: <em>export.php</em>. This legacy version of the export has the following limitation: the list of exported fields may vary depending on the output format and the data model of iTop. <br/>Should you want to garantee that the list of exported columns will remain stable on the long run, then you must specify a value for the attribute "Fields" and use the page <em>export-V2.php</em>.',
+	'UI:Query:UrlV1' => 'The list of fields has been left unspecified. The page <em>export-V2.php</em> cannot be invoked without this information. Therefore, the URL suggested here below points to the legacy page: <em>export.php</em>. This legacy version of the export has the following limitation: the list of exported fields may vary depending on the output format and the data model of iTop. <br/>Should you want to guarantee that the list of exported columns will remain stable on the long run, then you must specify a value for the attribute "Fields" and use the page <em>export-V2.php</em>.',
```
[the remark this answers is not in the document]
Comment (Molkobain) [reply dictionaries/en.dictionary.itop.ui.php]: Nevermind, I didn't saw the fixed typo.

[the remark this answers is not in the document]
Comment (jbostoen) [reply dictionaries/en.dictionary.itop.ui.php]: Typo in the English file. https://www.google.com/search?q=guarantee 
Probably an oversight from translating from French to English.

**Comment by Molkobain** [comment]
Merged. Thanks to both of you :)

**Comment by steffunky** [comment]
Hey, we found a typo in `nl.dictionary.itop.ui.php` and fixed it in 87f898d29dc093b9cb6af26b8f80cdb05c9b6899.

**Comment by piRGoif** [comment]
Hello,
We are terribly sorry, we missed 3 strings in the dictionnary files for the new TagSet v2.6 feature. I've just add them in commit 4072ed09acade8231c53006db8091be71144d633.
@jbostoen and @Hipska, would you mind translate those strings with a new PR ?
Sorry again, and many thanks.

**Pull request merged**